### PR TITLE
Feature/remove balance from integration

### PR DESCRIPTION
**Pull request by carlosshaka**
Create a function to remove balance from integration, fix tests and requires.

### CODE REVIEW

File: contracts/Ribon.sol [hunk 1]
```diff
@@ -61,28 +62,51 @@ contract Ribon {
...
+  }
 
+  function removeIntegrationBalance(address _integration, uint256 _amount)
+    public
+  {
     require(
-      donationPoolBalance - _amount >= 0,
-      "Donation pool balance should be bigger than 0"
+      msg.sender == integrationCouncil,
+      "Not on the integration council."
```
Comment (NickNish09) [inline contracts/Ribon.sol:96]: this require seems to be something that could appear on multiple methods. Is it possible to move this to another function? 
If this is too much of a work its ok

File: contracts/Ribon.sol [hunk 2]
```diff
@@ -106,4 +130,20 @@ contract Ribon {
 
     emit DonationAdded(_user, msg.sender, _nonProfit, _amount);
   }
+
+  function getIntegrationCouncil() public view returns (address) {
+    return integrationCouncil;
```
Comment (NickNish09) [inline contracts/Ribon.sol:135]: the integrationCouncil shouldnt be an array of addresses? Just to be sure

File: contracts/Ribon.sol (hunk 1 above)
Comment (carlosshaka) [reply contracts/Ribon.sol:96]: I searched about this and the best practice is to put the require directly in the same function

File: contracts/Ribon.sol (hunk 2 above)
Comment (carlosshaka) [reply contracts/Ribon.sol:135]: actually a multsig wallet is a wallet with a unique address, but it needs two or more to approve the transaction, this information about which wallets make up the multsig is in gnosis safe

**Review by NickNish09 — APPROVED** [review]

**Pull request merged**